armi/operators/tests/test_operators.py

```diff
@@ -123,19 +124,48 @@ def test_setStateToDefault(self):
         cs = self.o.setStateToDefault(self.o.cs)
         self.assertEqual(cs[CONF_RUN_TYPE], "Standard")
 
-    def test_snapshotRequest(self):
+    @patch("shutil.copy")
+    @patch("os.listdir")
```

Comment:
I think this is something not enough people do around here.  Nice!

Reply:
I started using `patch` after the last hackathon and it's been great!